src/domain/@shared/entity/entity.abstract.ts

```diff
@@ -1,13 +1,9 @@
 import Notification from "../notification/notification";
+
 export default abstract class Entity {
-  protected _id: string;
```

Comment:
eu removi o ID deste lugar porque não me parece fazer sentido com que todas as minhas entidades tenham este ID.
caso este ID for um inteiro dará problema, preferi remover daqui e deixar as opções em aberto